src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSourceRequest.java

```diff
@@ -187,6 +189,17 @@ protected BitbucketSCMSourceRequest(@NonNull final BitbucketSCMSource source,
         repoOwner = source.getRepoOwner();
         repository = source.getRepository();
         pullRequestData = new HashMap<>();
+        disableNotifications = context.notificationsDisabled();
+    }
+
+    /**
+     * Returns {@code true} if notifications are disabled.
+     *
+     * @return {@code true} if notifications are disabled.
+     */
+
+    public final boolean isDisableNotifications() {
+        return disableNotifications;
```

Comment:
Gratitious change, please create another PR

Reply:
This is actually already supported in a trait plugin: https://plugins.jenkins.io/skip-notifications-trait

Reply:
A list of available traits (for this plugin) would be helpful. Maybe just in the README.md?